receiver/stanzareceiver/converter.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stanzareceiver
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/observiq/stanza/entry"
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+func convert(obsLog *entry.Entry) pdata.Logs {
+	out := pdata.NewLogs()
+	logs := out.ResourceLogs()
+	logs.Resize(1)
+	rls := logs.At(0)
+
+	resource := rls.Resource()
+	resource.InitEmpty()
+	if len(obsLog.Resource) > 0 {
+		resourceAtts := resource.Attributes()
+		for k, v := range obsLog.Resource {
+			resourceAtts.InsertString(k, v)
+		}
+	}
+
+	rls.InstrumentationLibraryLogs().Resize(1)
```

Comment:
Would be nice to populate the InstrumentationLibrary object with the name and version of the stanza plugin used to generate the log.

Reply:
Good call. Done.

One note - stanza currently doesn't make its version available in-code, except when built as a standalone executable. I considered using `runtime/debug.ReadBuildInfo()`, but this seems a bit messy. For now, I've hardcoded the version, but we'll look into updating our release process to make this programmatically available.

Reply:
Ok, yeah knowing it is stanza and having the version hardcoded for now is good enough.